Object.java

```diff
@@ -5,17 +5,18 @@
 import java.io.ObjectInputStream;
 import java.io.ObjectOutputStream;
 
-public class Object {
-    public static Object deepClone(Object object){
+
+//The interface of the method, which allows a deep clone
+public interface deepClonanble {

```

Comment:
example of deep clone https://dzone.com/articles/shallow-and-deep-java-cloning

Interface name starts from Upercase letter

Reply:
This is not a deep clone